packages/api/src/controllers/animalController.js

```diff
@@ -51,86 +50,56 @@ const animalController = {
         const { farm_id } = req.headers;
         const result = [];
 
-        if (!Array.isArray(req.body)) {
-          await trx.rollback();
-          return res.status(400).send('Request body should be an array');
-        }
```

Comment:
In my humble opinion, this is nicer than an Unknown internal server error. You haven't removed the one from animalBatchController!

Reply:
Its more explicit yeah, for me I prefer to not feel like I have to handle every case where there should be an array provided it could add up to a lot of code.

The message provided by objections error "req.body is not iterable" makes enough sense to me. But I also think if you want to make a new Error Type in the future that makes this generic error more readable -- that is nice too!